examples/hackernews/public/components/Stories.js

```diff
@@ -0,0 +1,22 @@
+import { html } from "htm/preact";
+import { css } from "stylewars";
+import { useObservable, useDispatch } from "@depository/preact-hooks";
+import { Story } from "./Story.js";
+import { loadTopStories } from "../models/news.js";
+
+const styles = css``;
+
+export const Stories = () => {
+  const status = useObservable("statuses.topStories");
+  const topStories = useObservable("searches.topStories");
+  const dispatch = useDispatch();
+
+  if (!status) dispatch(loadTopStories());
```

Comment:
This is really interesting - because of the way hooks operate and you get the value come in after the first render you get a state change at mount time and thus effectively end up with an onmount.

Reply:
Yes it seems like a good replacement for useEffect for a lot of cases.